CMakeLists.txt

```diff
@@ -20,7 +24,7 @@ endif()
 
 include(CheckCXXCompilerFlag)
 
-if(NOT CMAKE_BUILD_TYPE)
+if(NOT CMAKE_BUIiLD_TYPE)
```

Comment:
This change can't be right.

Reply:
I was definitely using VIM or something when I did this hahaha.